docs/getting-started/building.md

```diff
@@ -52,26 +52,40 @@ This guide will demonstrate how to build a Zarhus OS image from zero!
 
 ## Build
 
-From `yocto` directory run:
+Depending on which features you want to have in your build, pass the desired
+`.yml` files via command line. You can read more on that in
+[kas documentation.](https://kas.readthedocs.io/en/latest/userguide/project-configuration.html#including-configuration-files-via-the-command-line)
+
+Then check BSP layers for available target platform (target platforms configs
+are located in `conf/machine` directory of every BSP layer) and choose one.
+Then, from `yocto` directory run:
 
 ```shell
-$ SHELL=/bin/bash kas-container build meta-zarhus/kas-IMAGE_TYPE.yml
+SHELL=/bin/bash KAS_MACHINE=<TARGET_NAME> kas-container build <KAS_FILES>
 ```
 
 !!! note
 
-    Replace `IMAGE_TYPE` with either `debug` or `prod`.
+    Replace `<TARGET_NAME>` with the name of the chosen target
+    configuration file, and `<KAS_FILES>` with a list of kas files, separated by
```

Comment:
Do we have explanation about available `<KAS_FILES>` and which ones are needed for what and in which order (e.g. debug needs to be after common)?
Maybe we should include section for those and link there?
And depending on whether `debug.yml` is included target filename might change e.g. `-debug` in `zarhus-base-image-debug-...` might be or not be there.

Reply:
Added here c7220ee098cd81c3616cb6975047eb14c31285ba. Other inf. user could get in kas documentation